README.md

```diff
@@ -71,27 +69,41 @@ python -m translator.source_code.icone_translator -i 'translator/sample files/ic
 ```
 python -m translator.source_code.cotrip_translator -i inputfile.json -o outputfile.geojson
 ```
+
 Example usage:
+
 ```
 python -m translator.source_code.cotrip_translator -i 'translator/sample files/cotrip_data/cotrip_1.json' 
 ```
 
-### Unit Testing
+### Execution for Combine_wzdx
+
+#### Step 1: Run the translator script (from Work_Zone/translator/source_code)
```

Comment:
Since there's only a single step, I don't think we need to call out "step 1..."

Reply:
should be fixed..i only did just put the word "step" instead of "step 1"

Reply:
```suggestion
#### Run the translator script (from Work_Zone/translator/source_code)
```

Reply:
i thought i changed it but i guess i did not ..it should be fixed now

Reply:
Can we still remove the "step: " text? It reads quite awkward.

Reply:
It should be fixed now.